h2o-py/h2o/model/extensions/predict_contributions.py

```diff
@@ -0,0 +1,50 @@
+import h2o
+from h2o.exceptions import H2OValueError
+from h2o.job import H2OJob
+from h2o.utils.typechecks import assert_is_type, Enum
+
+
+class PredictContributions:
```

Comment:
I'd rename this file+class to just `contributions.py` or `shapley.py` (resp. `Contributions` or `Shapley`): idea behind those extensions is to group methods by features, if one algo supports Shapley values and if we add more methods around those outside just returning them as a frame like in `_predict_contributions`, then those additional methods should appear in the same extension.

Reply:
no problem, will do